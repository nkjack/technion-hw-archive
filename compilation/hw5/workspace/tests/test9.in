void maxTestsBigTruncationTest() {
	int t = 126b + 224b + 139 - 141b - 225b;
	printi(t);
	print("\n");
	int s = 126b + 224b + 139b - 141b - 225b;
	printi(s);
	print("\n");
}

void foo(int x,int y)
@pre(x < 0 or x > 10 and (x + y) < 20)
@pre(x > 0)
{
	maxTestsBigTruncationTest();
}

void main() {
     foo(12,1);
     print("GOOD TEST\n");
}
