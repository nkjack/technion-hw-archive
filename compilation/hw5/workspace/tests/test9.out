Loaded: ./exceptions.s
-133
123
GOOD TEST
